ocs_ci/ocs/exceptions.py

```diff
@@ -14,6 +14,14 @@ class CephHealthException(Exception):
     pass
 
 
+class CephHealthExceptionRecovered(Exception):
+    pass
+
+
+class CephHealthExceptionNotRecovered(Exception):
+    pass
```

Comment:
We didn't have any hierarchy used in the exceptions yet, but wouldn't make sense to place those two new exceptions under some covering one? (Maybe`CephHealthException`, if it will not break anything elsewhere?)

Reply:
I need to distinguish between those two specific cases and act differently than we experience CephHealthException, hence I cannot inherit from it and make some hierarchy